### DISCUSSION TITLE
Milestone Transition Statements [1/2]

### DISCUSSION THREAD

**Discussion by lex939** [uprm-inso4116-2024-2025-s1/semester-project-SafeRUM#173] [Key]
Hello everyone! The following discussion is available for all project members to discuss any changes they might want to make while the project transitions from Milestone 1 to Milestone 2. This discussion will close on Friday, September 27. 

_**Please note**: The next (and last) discussion for any milestone transition statements from group members will be posted around the time Milestone 2 ends. The discussion will later be available for all project members to discuss any changes they might want to make while the project transitions from Milestone 2 to Milestone 3._

**Comment by JorgeRivera94** [comment]
Hello!

Seeing as the purpose of the report system is to provide a quick and easy way for users to submit reports, the report categories should be relevant to common and relevant situations that already arise in campus with some frequency. 

The proposed flow in #171 is meant to speed up the reporting process, minimizing the reporting time so the user can focus on their safety. For this reason, the initial category selection should reflect already common and time sensitive situations.

I would like to request approval to create an issue for Trust System Team to evaluate what these situations are and how many labels should be available for the user to choose from, keeping in mind that the user's attention should not be divided in to many options.

I suggest exploring the criminal acts in "Informe de Actos Delictivos" emails to identify criminal acts occurring in our campus and their frequency.

**Reply by lex939** [reply]
@evnd04 @YanehRuiz Please review.

**Comment by lex939** [comment]
Posting here to remind everyone that this discussion closes today. If there are any other statements you and/or your peers would like to make, please do them by 8pm. Thank you.

**Comment by lex939** [comment]
This discussion is now closed and locked.

_**Please note:** The next (and last) discussion for any milestone transition statements from group members will be posted around the time Milestone 2 ends. That discussion will be available around that time for all project members to discuss any changes they might want to make while the project transitions from Milestone 2 to Milestone 3._
